mewarpx/mewarpx/setups_store/diode_setup.py

```diff
@@ -594,15 +603,17 @@ def init_fluxdiag(self):
 
     def init_field_diag(self):
         print('### Init Diode FieldDiag ###')
-        diagnostic_intervals = f"::{self.DIAG_STEPS}"
-        self.field_diag = picmi.FieldDiagnostic(
+
+        diag = field_diagnostic.FieldDiagnostic(
```

Comment:
Either save it to a member variable `self.field_diag` or don't save it at all, I think either one is OK for now, but assigning it to diag doesn't do anything.
```suggestion
        field_diagnostic.FieldDiagnostic(
```